data/json/items/ammo/4570.json

```
@@ -0,0 +1,50 @@
+[
+  { "id": "4570_sp",
+    "type": "AMMO",
+    "name": ".45-70 SP",
+    "description": ".45-70 Government ammunition loaded with a 305 grain soft point round.  One of the oldest cartridges still in use, it is still a favorite for large game hunting at short ranges.",
+    "weight": 35,
+    "volume": 1,
+    "price": 4000,
+    "material": [ "brass", "powder" ],
+    "symbol": "+",
+    "color": "green",
+    "count": 20,
+    "stack_size": 20,
+    "ammo_type": "4570",
+    "casing": "4570_casing",
+    "range": 48,
+    "damage": 66,
+    "pierce": 3,
+    "dispersion": 20,
+    "recoil": 4300,
+    "effects": [ "COOKOFF" ]
+  },
+  { "id": "4570_pen",
+    "copy-from": "4570_sp",
+    "type": "AMMO",
+    "name": ".45-70 +P Penetrator",
+    "description": ".45-70 Government +P ammunition loaded with a 305 grain solid copper penetrator projectile.  Designed for maximum penetration through thick hide and bone while maintaining ideal wounding characteristics.",
+    "price": 5000,
+    "count": 10,
+    "relative": { "range": 2, "damage": 2, "pierce": 8 }
+  },
+  { "id": "reloaded_4570_sp",
+    "copy-from": "4570_sp",
+    "type": "AMMO",
+    "name": "Reloaded .45-70 SP",
```

Capital letter not needed here, to be the same as other reloaded ammo.
```suggestion
    "name": "reloaded .45-70 SP",
```